### PR TITLE
image: Add script to get changes in image

### PR DESCRIPTION
**Pull request by jcvenegas**
Add script to autogenerate a list of changes between images.

Fixes: #141

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

**Comment by jcvenegas** [comment]
@jodh-intel  @gorozco1  @erick0z , can you take a looks to this script ?.

**Comment by gorozco1** [comment]
@jcvenegas please add a sample of the output

**Comment by jcvenegas** [comment]
example with with version `17270` in `versions.txt` to `18280`
```bash
$ ./get-image-changes.sh 18280 > changes

$ cat changes
version: 17490
Changes in package clear-containers-agent (from 2651b6af67e61460407cf9ec403d8260e81acc36-8 to 118c59f374bbe266c897b2a368be3774e7b1f585-9):
     Jose Carlos Venegas Munoz - version bump from 118c59f374bbe266c897b2a368be3774e7b1f585-8 to 118c59f374bbe266c897b2a368be3774e7b1f585-9
     Jose Carlos Venegas Munoz - New Clear Containers Agent version 118c59f
https://download.clearlinux.org/releases/17490/clear/RELEASENOTES
version: 17510
Changes in package coreutils (from 8.27-40 to 8.28-41):
     Arjan van de Ven - coreutils: Autospec creation for update from version 8.27 to version 8.28
https://download.clearlinux.org/releases/17510/clear/RELEASENOTES
version: 17880
Changes in package clear-containers-agent (from 118c59f374bbe266c897b2a368be3774e7b1f585-9 to 0.1.0-10):
     Jose Carlos Venegas Munoz - version bump from 0.1.0-9 to 0.1.0-10
     Jose Carlos Venegas Munoz - New Clear Containers Agent version 0.1.0
https://download.clearlinux.org/releases/17880/clear/RELEASENOTES
version: 17970
Changes in package systemd (from 234-149 to 234-150):
     Arjan van de Ven - version bump from 234-149 to 234-150
https://download.clearlinux.org/releases/17970/clear/RELEASENOTES
version: 18140
Changes in package clear-containers-agent (from 0.1.0-10 to 0.1.0-11):
     Arjan van de Ven - version bump from 0.1.0-10 to 0.1.0-11
https://download.clearlinux.org/releases/18140/clear/RELEASENOTES
version: 18220
Changes in package clear-containers-agent (from 0.1.0-11 to 84761a194a72018c67f1fa8a6a1267b8ea120f82-12):
     Jose Carlos Venegas Munoz - version bump from 84761a194a72018c67f1fa8a6a1267b8ea120f82-11 to 84761a194a72018c67f1fa8a6a1267b8ea120f82-12
     Jose Carlos Venegas Munoz - new version: agent version 84761a1
https://download.clearlinux.org/releases/18220/clear/RELEASENOTES
version: 18280
Changes in package iptables (from 1.6.1-20 to 1.6.1-21):
     George T Kramer - iptables: Autospec creation for version 1.6.1
https://download.clearlinux.org/releases/18280/clear/RELEASENOTES

```

**Review by jodh-intel — APPROVED** [review]
Nice!!

lgtm.

A few random thoughts:

- Obviously, the output will require a manual review in case the clr release notes broke / changed format some how (it's probably worth notifying the appropriate team that we are "consuming" the release notes and to keep us informed if they plan to change anything.
- Wouldn't it be great if clr release notes were in YAML format? :smile:
- If a clr release is withdrawn, this script will still show the details from it. I'm not entirely clear if that's a problem or not since the next release should resolve any issues with the withdrawn release and (I think) builds on the broken release anyway. Either way, having this output is going to be a huge help in understanding "what changed" so thanks!

**Comment by jcvenegas** [comment]
@jodh-intel yeah, YAML would be great format for those releases. I will ask to Clear Linux ML what they think about it. 

If the release withdrawn, I think we still want to know what changes are done in the packages, if  there withdrawn release CL dont  revert changes but fix them in the nexts releases (numbers are cheap)

**Comment by gorozco1** [comment]
lgtm

**Review by gorozco1 — APPROVED** [review]

**Review by gorozco1 — CHANGES_REQUESTED** [review]
@jcvenegas could you change the location of this script to packaging/scripts

**Review by gorozco1 — CHANGES_REQUESTED** [review]
And please update the README under packaging/scripts.

File: scripts/README.md [hunk 1]
```diff
@@ -15,3 +15,22 @@ Example usage:
 ```
   $ configure-hypervisor.sh qemu-lite
 ```
+
+## get-image-changes.sh
```
Comment (gorozco1) [inline scripts/README.md]: ## `get-image-changes.sh`

File: scripts/README.md [hunk 2]
```diff
@@ -15,3 +15,22 @@ Example usage:
...
+
+This scripts generates the list of changed packages that are used in Clear
```
Comment (gorozco1) [inline scripts/README.md]: This script

**Review by gorozco1 — CHANGES_REQUESTED** [review]
minor style changes requested.

**Review by gorozco1 — APPROVED** [review]

**Pull request merged**